packages/x-adapter-platform/src/schemas/responses/semantic-queries-response.schema.ts

```diff
@@ -0,0 +1,19 @@
+import { createMutableSchema, Schema } from '@empathyco/x-adapter';
+import { SemanticQueriesResponse } from '@empathyco/x-types';
+import { semanticQuerySchema } from '../models/semantic-query.schema';
+// eslint-disable-next-line max-len
+import { PlatformSemanticQueriesResponse } from '../../types/responses/semantic-queries-response.model';
+
+/**
+ * Default implementation for the RelatedTagsResponseSchema.
+ *
+ * @public
+ */
+export const semanticQueriesResponseSchema = createMutableSchema(<
+  Schema<PlatformSemanticQueriesResponse, SemanticQueriesResponse>
+>{
+  semanticQueries: {
```

Comment:
We're not mapping the `msg` anywhere. [It only appears in case of error](https://searchbroker.atlassian.net/wiki/spaces/EAE/pages/4020011011/Semantics+API#Response), I don't know if sending a msg in case of error is something exclusive for this endpoint's backend or we are handling that in other adapters

Reply:
I would say that we don't need that error message.
We don't show back error messages in the other endpoints, and, even if they return that error, we don't know if it is a user-friendly message (probably not).
Also, the worst that can happen if there is some error is that no semantic queries get rendered.